packages/jellyfish-api-core/__tests__/category/oracle/listFixedIntervalPrices.test.ts

```diff
@@ -0,0 +1,263 @@
+      fixedIntervalPriceId: 'UBER/USD'
+    })
+    await testing.generate(1)
+  }
+
+  it('should listFixedIntervalPrices', async () => {
+    {
+      const prices = await testing.rpc.oracle.listFixedIntervalPrices()
+      expect(prices).toStrictEqual([
+        {
+          priceFeedId: 'BTC/USD',
+          activePrice: new BigNumber('60000'),
+          nextPrice: new BigNumber('60000'),
+          timestamp: expect.any(Number),
+          isLive: true
+        },
+        {
+          priceFeedId: 'DFI/USD',
+          activePrice: new BigNumber('1'),
+          nextPrice: new BigNumber('1'),
+          timestamp: expect.any(Number),
+          isLive: true
+        },
+        {
+          priceFeedId: 'TSLA/USD',
+          activePrice: new BigNumber('0'),
+          nextPrice: new BigNumber('2'),
+          timestamp: expect.any(Number),
+          isLive: false
+        },
+        {
+          priceFeedId: 'UBER/USD',
+          activePrice: new BigNumber('0'),
+          nextPrice: new BigNumber('8'),
+          timestamp: expect.any(Number),
+          isLive: false
+        }
+      ])
+    }
+
+    {
+      await testing.generate(6)
+      const prices = await testing.rpc.oracle.listFixedIntervalPrices()
+      expect(prices[3]).toStrictEqual({
+        priceFeedId: 'UBER/USD',
+        activePrice: new BigNumber('8'),
+        nextPrice: new BigNumber('8'),
+        timestamp: expect.any(Number),
+        isLive: true
+      })
+    }
+
+    {
+      await testing.rpc.oracle.setOracleData(oracleId, timestamp, { prices: [{ tokenAmount: '32@UBER', currency: 'USD' }] })
+      const prices = await testing.rpc.oracle.listFixedIntervalPrices()
+      expect(prices[3]).toStrictEqual({
+        priceFeedId: 'UBER/USD',
+        activePrice: new BigNumber('8'),
+        nextPrice: new BigNumber('8'),
+        timestamp: expect.any(Number),
+        isLive: true
+      })
+    }
+
+    {
+      await testing.generate(6)
+      const prices = await testing.rpc.oracle.listFixedIntervalPrices()
+      expect(prices[3]).toStrictEqual({
+        priceFeedId: 'UBER/USD',
+        activePrice: new BigNumber('8'),
+        nextPrice: new BigNumber('32'),
+        timestamp: expect.any(Number),
+        isLive: false
+      })
+    }
+
+    {
+      await testing.generate(6)
+      const prices = await testing.rpc.oracle.listFixedIntervalPrices()
+      expect(prices[3]).toStrictEqual({
+        priceFeedId: 'UBER/USD',
+        activePrice: new BigNumber('32'),
+        nextPrice: new BigNumber('32'),
+        timestamp: expect.any(Number),
+        isLive: true
+      })
+    }
+  })
+
+  it('should listFixedIntervalPrices with limit', async () => {
+    const prices = await testing.rpc.oracle.listFixedIntervalPrices({ limit: 1 })
+    expect(prices.length).toStrictEqual(1) // [0] is { activePriceBlock, nextPriceBlock } then priceInfo
+  })
+
+  it('should listFixedIntervalPrices with pagination start and including_start', async () => {
```

Comment:
test content does not match the description/name🤔
if want to test with `pagination start and including_start`, should pass those parameters and check for the length of the return array.

Reply:
ohh.. normally pagination with start param the including_start will be auto be true.. 
but just found out this pagination param don't include including_start :sweat_smile: 
removed